connectors/src/connectors/zendesk/lib/help_center_permissions.ts

```diff
@@ -169,6 +169,8 @@ export async function allowSyncZendeskCategory({
     }
   }
 
+  await allowSyncZendeskHelpCenter({ connectorId, connectionId, brandId });
```

Comment:
If we do that we still sync only the single category that was allowed in the help center?

Reply:
yes!